### PR TITLE
Upgrade SKY UX 3

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,45 @@
...
+    "@angular/platform-browser": "7.2.15",
+    "@angular/platform-browser-dynamic": "7.2.15",
+    "@angular/router": "7.2.15",
+    "@blackbaud/auth-client": "2.15.1",
     "@blackbaud/skyux-builder-plugin-code-block": "1.1.0",
-    "@blackbaud/skyux-builder": "1.35.0",
     "@blackbaud/skyux-lib-clipboard": "1.1.0",
-    "@blackbaud/skyux": "2.51.1",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0"
+    "@blackbaud/skyux-lib-code-block": "1.2.0",
```
Comment (Blackbaud-BrandonJones) [inline package.json]: Is this depending on itself?

Comment (Blackbaud-SteveBrush) [reply package.json]: Good catch. Fixed.

File: .gitignore [hunk 2]
```diff
@@ -30,5 +30,4 @@ dist
 
 *.ngfactory.ts
 *.ngsummary.json
-yarn.lock
-package-lock.json
+yarn.lock
```
Comment (Blackbaud-BrandonJones) [inline .gitignore]: Is committing the package-lock.json recommended practice now?

Comment (Blackbaud-SteveBrush) [reply .gitignore]: Yep!

Comment (Blackbaud-BrandonJones) [reply .gitignore]: 👍

**Pull request merged**